### DISCUSSION TITLE
Karoo app

### DISCUSSION THREAD

**Discussion by gloscherrybomb** [maca88/SmartBikeLights#80] [General]
Hi, now Hammerhead has announced a Karoo SDK, would you be interested in a SBL app for the Karoo? If not maybe I could have a crack?

**Comment by maca88** [comment]
Hi, 

> would you be interested in a SBL app for the Karoo?

I’m not currently interested in porting SBL to Karoo, as I don’t own a Karoo device and am satisfied with my Garmin Edge. I’m in no rush to switch.

> If not maybe I could have a crack?

Of course, feel free to use whatever code you find useful from this repository. You don't need my permission anyway, this repo is under MIT license, so you can do whatever you want with the code :)

I checked the new Karoo Extensions out of my curiosity and based on the current state, it doesn't seem to support ANT+ lights. I am assuming this based on the DataType definition: https://github.com/hammerheadnav/karoo-ext/blob/efac75b5f1b1fdf9f792eeffbf2dec92e48cdd65/lib/src/main/kotlin/io/hammerhead/karooext/models/DataType.kt#L37

If that is the case, then I would recommend you to open a feature request for adding support for ANT+ lights.
